charts/operator/values.go

```diff
@@ -0,0 +1,154 @@
+	ClusterDomain           string                        `json:"clusterDomain"`
+	Image                   Image                         `json:"image"`
+	KubeRBACProxy           KubeRBACProxyConfig           `json:"kubeRbacProxy"`
+	Configurator            Image                         `json:"configurator"`
+	Config                  Config                        `json:"config"`
+	ImagePullSecrets        []corev1.LocalObjectReference `json:"imagePullSecrets"`
+	LogLevel                string                        `json:"logLevel"`
+	RBAC                    RBAC                          `json:"rbac"`
+	Webhook                 Webhook                       `json:"webhook"`
+	ServiceAccount          ServiceAccountConfig          `json:"serviceAccount"`
+	Resources               corev1.ResourceRequirements   `json:"resources"`
+	NodeSelector            map[string]string             `json:"nodeSelector"`
+	Tolerations             []corev1.Toleration           `json:"tolerations"`
+	Affinity                WrappedAffinity               `json:"affinity" jsonschema:"deprecated"`
+	Strategy                *appsv1.DeploymentStrategy    `json:"strategy,omitempty"`
+	Annotations             map[string]string             `json:"annotations,omitempty"`
+	PodAnnotations          map[string]string             `json:"podAnnotations"`
+	PodLabels               map[string]string             `json:"podLabels"`
+	AdditionalCmdFlags      []string                      `json:"additionalCmdFlags"`
+	CommonLabels            map[string]string             `json:"commonLabels"`
+	Monitoring              MonitoringConfig              `json:"monitoring"`
+	WebhookSecretName       string                        `json:"webhookSecretName"`
+	PodSpec                 *WrappedPodSpec               `json:"podSpec,omitempty"`
+	OperatorSecurityContext *corev1.SecurityContext       `json:"operatorSecurityContext,omitempty"`
+	LivenessProbe           *corev1.Probe                 `json:"livenessProbe,omitempty"`
+	ReadinessProbe          *corev1.Probe                 `json:"readinessProbe,omitempty"`
+	Scope                   OperatorScope                 `json:"scope" jsonschema:"required,pattern=^(Namespace|Cluster)$,description=Sets the scope of the Redpanda Operator."`
+}
+
+type WrappedAffinity corev1.Affinity
+
+func (a *WrappedAffinity) ToCoreV1Affinity() *corev1.Affinity {
+	return &corev1.Affinity{
+		NodeAffinity:    a.NodeAffinity,
+		PodAffinity:     a.PodAffinity,
+		PodAntiAffinity: a.PodAntiAffinity,
+	}
+}
+
+// +gotohelm:ignore=true
+func (WrappedAffinity) JSONSchemaExtend(schema *jsonschema.Schema) {
+	makeArrayNullableRecursive(schema)
+}
+
+type WrappedPodSpec corev1.PodSpec
+
+// +gotohelm:ignore=true
+func (WrappedPodSpec) JSONSchemaExtend(schema *jsonschema.Schema) {
+	makeArrayNullableRecursive(schema)
+}
+
+// +gotohelm:ignore=true
+func makeArrayNullableRecursive(schema *jsonschema.Schema) {
```

Comment:
Would it make sense to just run this against the final generated schema in `genschema`?